Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -56,14 +56,14 @@ import BraintreeCore
 }
 
 /// Options for the PayPal Checkout flow.
-@objcMembers public class BTPayPalCheckoutRequest: BTPayPalRequest {
+@objcMembers open class BTPayPalCheckoutRequest: BTPayPalRequest {
```

Comment:
This is now needs to be `open` because we are subclassing it outside of it's defining module